SleepEngine/Engine/GameTemplate/Game.cpp

```diff
@@ -29,11 +33,14 @@ Game::Game(size_t width, size_t height)
 
     if (m_instance)
     {
-        LOG_AND_ASSERT_ERROR(false, "Attempt to create second game window! It is forbidden!");
-        return;
+	    LOG_AND_ASSERT_ERROR(false, "Attempt to create second game! It is forbidden!");
+	    return;
```

Comment:
Please, don't use tabs here.
If you want you can leave it like this now, but later we should run all code through Clang Format